ci-operator/config/openshift/loki/openshift-loki-release-5.5.yaml

```diff
@@ -65,6 +65,7 @@ tests:
         requests:
           cpu: 100m
           memory: 200Mi
+      timeout: 10m0s
```

Comment:
This is the wrong timeout. It needs an option to `golangci-lint` itself. See [here](https://github.com/openshift/cluster-logging-operator/pull/1644).